Complete sprint 8

Evo-Eclipse opened the pull request:
Storyboard and View Updates:
- Removed the old storyboard configuration.
- Added a new storyboard configuration for the `ImagesListViewController` with a table view and custom cells.

New Components:
- Introduced a new `GradientView` class that uses a `CAGradientLayer` to display a gradient background.
- Added a new `ImagesListCell` class for custom table view cells, including outlets for an image, like button, and date label.
- Created a new `ImagesListViewController` class that manages the display of a list of images in a table view, including configuring cells and handling table view delegate and data source methods.

Project Configuration:
- Updated the project configuration to reflect the new location of the `Info.plist` file.

Evo-Eclipse committed:
Complete sprint 8

- Completed layout for the Launch Screen
- Added and designed the image feed screen according to the provided mockups

Kirchberg commented on ImageFeed/Presentation/GradientView.swift:
```diff
@@ -0,0 +1,30 @@
+
+import UIKit
+
+class GradientView: UIView {
```
Давай проставлять ключевое слово `final` классам для ускорения работы компилятора.
Это нужно для оптимизация работы компилятора: ты явно сообщаешь о том, что класс не планируется расширять наследованием. 
В таком случае компилятор будет быстрее находить адрес ячейки памяти при вызове функции.

Эта тема про диспетчеризацию, если интересно, можно почитать здесь:
https://habr.com/ru/post/673400/
https://habr.com/ru/company/simbirsoft/blog/673636/

Kirchberg reviewed (approved):
⚠️ Критические замечания (это нужно исправить обязательно):

1. В настройках проекта выставлена минимальная версия iOS 15.0, по условию задания необходимо выставить либо 13.0, либо 14.0, [см. криншот проекта](https://ibb.co/yBNLSZPr).
2. Проект не запустился, так как не был найден Info.plist, [см. скриншот ошибки](https://ibb.co/bRBr89Xs). Для того, чтобы это исправить необходимо файл Info.plist разместить в корневой директории, [см. скриншот](https://ibb.co/x88nFJGn).

Без исправлений этих пунктов, я не смогу принять у тебя работу. Если возникнут вопросы, обязательно задавай их Наставнику!

Не могу не отметить, что у тебя получилась очень аккуратная и классная работа, молодец ✨

Evo-Eclipse committed:
Review fixes

- Set minimum iOS version to 13.0
- Fixed Info.plist path in build settings
- Smoothed gradient view

Evo-Eclipse commented:
Project Configuration:
- Set minimum iOS version to 13.0 in `IPHONEOS_DEPLOYMENT_TARGET` to meet project requirements.
- Updated `Info.plist` file path in `Target` `Build Settings` to resolve project build issues.

UI Improvements:
- Enhanced `GradientView.swift` with a smoother gradient transition using refined color stops and alpha values.

Evo-Eclipse commented:
Fixed invalid commit name.

Kirchberg reviewed (approved):
Не забудь вмерджить пул-реквест в основную ветку.
Успехов в следующем задании!

Evo-Eclipse commented:
Thank you for your review and interesting articles 😺

Evo-Eclipse committed:
Remove Unnecessary Comments
